### PR TITLE
Adding best practices

### PR DESCRIPTION
**Pull request by Nyholm**
| Q             | A
| ------------- | ---
| License       | MIT

This PR is meant to add guides, best practices, or "rules of submissions" to the contrib recipes. These are the most common issues I've seen with recipes. 

These bullets probably needs to be reworded, but here is the gist.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -34,3 +34,21 @@ automatically merged when the following conditions are met (in that order):
...
 .. _`documentation`: https://github.com/symfony/recipes
+
+Best practices
+--------------
+
+1. A recipe for a bundle should not contain all the default config. A good
+recipe only contains suggestion config for an application. That is config that needs
+to be configured but no real default value exists.
+
+2. No recipe should modify other bundle's configuration. The only exception is when
```
Comment (nicolas-grekas) [inline README.rst:45]: s/append/appends

File: README.rst [hunk 2]
```diff
@@ -34,3 +34,21 @@ automatically merged when the following conditions are met (in that order):
...
+a recipe append to a collection.
+
+3. No config should be "copied from package". That makes the recipes impossible
+to maintain and to assure their quality.
+
+4. Most Symfony bundles do not need a recipe. A bundle without config or routes
+will be installed by Flex with no recipe required.
```
Comment (nicolas-grekas) [inline README.rst:52]: missing indentation on all next-lines :)

**Comment by Nyholm** [comment]
Thank you for the review. I've reworded the PR and introduced headings instead of the bullet list

**Review by nicolas-grekas — APPROVED** [review]

**Comment by Nyholm** [comment]
Thank you

**Comment by nicolas-grekas** [comment]
We could inline the remaining recipes that use copy-from-package maybe?
Another idea: provide a GitHub Action that submit PRs to this repo when a package changes some of its default config in its own repo.

**Pull request merged**